emcc.py

```diff
@@ -613,6 +614,11 @@ def get_binaryen_passes():
     passes += ['--safe-heap']
   if settings.MEMORY64 == 2:
     passes += ['--memory64-lowering']
+  # sign-ext is enabled by default by llvm.  If the target browser settings don't support
+  # this we lower it away here using a binaryen pass.
+  if not feature_matrix.caniuse(feature_matrix.Feature.SIGN_EXT):
+    logger.debug('lowering sign-ext proposal due to incompatiable target browser engines')
```

Comment:
`incompatiable` -> `incompatible`

Reply:
Also if I were nitpicking i'd call this a "feature" rather than a "proposal" since it's fully merged into the spec :D